Cabal/Distribution/PackageDescription/Parsec.hs

```diff
@@ -185,7 +187,8 @@ parseGenericPackageDescription' fs = do
         | name == "executable" = do
             -- TODO: make a combinator for this, repeated in tests and bench
             name' <- parseName pos args
-            exe <- parseCondTree executableFieldDescrs storeXFieldsExe (targetBuildDepends . buildInfo) emptyExecutable fields
+            -- Note: we don't parse the "executable" field here, hence the tail hack. Duncan 2010
```

Comment:
@dcoutts the same note is in `D.PD.Parse.hs` (from 2010), I have no idea why `executable:` field is dropped?